### DISCUSSION TITLE
How to add apps to other workspaces?

### DISCUSSION THREAD

**Discussion by danddinh** [ToolJet/ToolJet#6543] [Q&A] [answered]
Similar to users, apps can be added to multiple user groups.

I have different workspaces, how to **add apps to multiple workspaces** just like adding apps to user groups?

There's work-around to export app and import app after changing workspace but it's clumpsy.

**Comment by withshubh** [comment, marked as answer]
Hey @dandeancook - Currently, export-import is the only way to share apps across different workspaces.
